velox/exec/HashProbe.cpp

```diff
@@ -481,11 +482,19 @@ RowVectorPtr HashProbe::getOutput() {
           ++numOut;
         }
       }
+    } else if (isAntiJoin(joinType_) && !filter_) {
```

Comment:
Can this branch be combined with the previous branch?

Reply:
Not easily, the null condition is inverted.